openfga/v1/authzmodel.proto

```diff
@@ -7,58 +7,61 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 import "validate/validate.proto";
 
 message AuthorizationModel {
-  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
-    example: "{\"id\": \"01G5JAVJ41T49E9TT3SKVS7X1J\", \"type_definitions\":[{\"type\":\"document\",\"relations\":{\"reader\":{\"union\":{\"child\":[{\"this\":{}},{\"computedUserset\":{\"object\":\"\",\"relation\":\"writer\"}}]}},\"writer\":{\"this\":{}}}}]}"
-  };
   string id = 1 [
-    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      example: "\"01G5JAVJ41T49E9TT3SKVS7X1J\""
-    },
-    (validate.rules).string = {
-      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$"
-    }
+    (validate.rules).string = { pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$" },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = { example: '"01G5JAVJ41T49E9TT3SKVS7X1J"' }
   ];
-  repeated TypeDefinition type_definitions = 2 [json_name = "type_definitions"];
-}
 
-message TypeDefinitions {
-  repeated TypeDefinition type_definitions = 1 [
+  repeated TypeDefinition type_definitions = 2 [
     json_name = "type_definitions",
-
-    (validate.rules).repeated = {
-      min_items: 1
-    },
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      minimum: 1,
+      example: '[{"type": "user"}, {"type":"document","relations":{"reader":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"writer"}}]}},"writer":{"this":{}}},"metadata":{"relations":{"reader":{"directly_related_user_types":[{"type":"user"}]},"writer":{"directly_related_user_types":[{"type":"user"}]}}}}]'
```

Comment:
```suggestion
      example: '[{"type": "user", "relations": {}}, {"type":"document","relations":{"reader":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"writer"}}]}},"writer":{"this":{}}},"metadata":{"relations":{"reader":{"directly_related_user_types":[{"type":"user"}]},"writer":{"directly_related_user_types":[{"type":"user"}]}}}}]'
```

Reply:
Internally we handle the absence of `relations` so it doesn't need to be there. Do you want it in the example?

Reply:
Internally, you may accept it, but in [the protos](https://github.com/openfga/api/pull/27/files#diff-4fff4cf94235ef3370ef32783dc14676cf70fc97cf96d3a3c13fe6bbcf018683R32), it is marked as required. Which means it will be invalid if clients try to send it.

Let's update the example to match the protos, or make it not required in the protos.

Reply:
Ooooohhh! Good catch. Removed the required. Thank you!